tools/cli/database_mock.go

```diff
@@ -34,33 +34,34 @@ import (
 
 	persistence "github.com/uber/cadence/common/persistence"
 	client "github.com/uber/cadence/common/persistence/client"
+	invariant "github.com/uber/cadence/common/reconciliation/invariant"
 )
 
-// MockPersistenceManagerFactory is a mock of PersistenceManagerFactory interface.
-type MockPersistenceManagerFactory struct {
+// MockManagerFactory is a mock of ManagerFactory interface.
```

Comment:
nit suggestion:
Explicitly provide the Interface name in mockgen. The existing solution will generate mocks for all interfaces in this package, which might be unexpected.